Kitodo/src/main/java/org/kitodo/production/forms/CommentForm.java

```diff
@@ -238,67 +235,37 @@ public String solveProblemForAllBatchProcesses(Comment comment) {
         return MessageFormat.format(REDIRECT_PATH, "tasks");
     }
 
-    /**
-     * Get process.
-     *
-     * @return value of process
-     */
-    public Process getProcess() {
-        return process;
-    }
-
-    /**
-     * Set process.
-     *
-     * @param process as org.kitodo.data.database.beans.Process
-     */
-    public void setProcess(Process process) {
-        this.process = process;
-    }
-
     /**
      * refresh the process in the session.
      *
      * @param process Object process to refresh
      */
-    public void refreshProcess(Process process) {
+    private void refreshProcess(Process process) {
```

Comment:
Is the refresh of process still needed if this form actually operates now on task object?

Reply:
I think this is still required because `getAllComments` uses the task's process to convert all old comments (e.g. "WikiField") into the new format. But I would prefer if @IkramMaalej could clear this up.

Reply:
This method is still required because we need to refresh the process after every reporting and solving a problem.